ospd_openvas/daemon.py

```diff
@@ -1094,7 +1099,7 @@ def exec_scan(self, scan_id: str):
         do_not_launch = False
         kbdb = self.main_db.get_new_kb_database()
         scan_prefs = PreferenceHandler(
-            scan_id, kbdb, self.scan_collection, self.nvti
+            scan_id, kbdb, self.scan_collection, self.nvti, self.notus.exists
```

Comment:
`self.notus` might be `None`

Reply:
We are passing a method to check if an oid is in the notus cache (not the notus object). 
Unfortunately, gvmd does not use the group filter for sending entire families of plugins (group:"Ubuntu Local Security Check), but it is passing each single OID in the plugins list.
This means that we need to check if the OID belongs to the notus cache or not, to excluded it. So, it is not sent to openvas as single plugin. The method notus.exists(OID) checks that.

LSC are run for all plugins. IMHO the clients should send the family name instead of list of OIDs. In case of receiving a the family name for a LSC, we could remove all notus stuff from the preferencehandler. But it is not the case, and I wouldn't expect such a change in the gvmd side.

Check the need and usage of the method [here](https://github.com/greenbone/ospd-openvas/blob/075bd851fc8dc0488d35364929f8f03c61a307a1/ospd_openvas/preferencehandler.py#L110) and [here](https://github.com/greenbone/ospd-openvas/blob/075bd851fc8dc0488d35364929f8f03c61a307a1/ospd_openvas/preferencehandler.py#L251)